components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/constants/SQLConstants.java

```diff
@@ -68,6 +68,27 @@ public class SQLConstants {
             "DELETE FROM IDN_NOTIFICATION_ORG_TEMPLATE WHERE TYPE_ID = (" + GET_NOTIFICATION_TYPE_ID_SQL +
                     ") AND TENANT_ID = :TENANT_ID;";
 
+    // sql constants for org notification template with unicode datatypes
+    public static final String INSERT_ORG_NOTIFICATION_TEMPLATE_SQL_UNICODE =
+            "INSERT INTO IDN_NOTIFICATION_ORG_TEMPLATE " +
+                    "(TEMPLATE_KEY, LOCALE, NSUBJECT, NBODY, NFOOTER, CONTENT_TYPE, TYPE_ID, TENANT_ID) " +
```

Comment:
Are these column names correct? `NSUBJECT, NBODY, NFOOTER`
Shouldn't they be changed to `SUBJECT, BODY, FOOTER` instead?